package.json

```diff
@@ -119,7 +119,7 @@
     "typescript": "^3.9.5"
   },
   "peerDependencies": {
-    "stylelint-copyright": "^2.0.0"
+    "stylelint-copyright": "2.0.0-alpha.66"
```

Comment:
Hi @lex111 

@Simek noticed that. this version didn't get bumped to alpha 68 with latest release, I believe it's because the root package is not a monorepo published package.

Is there a good reason for this change or should we revert?

Reply:
I see, I didn’t know about it, then we will get the same warning, although it’s probably better than what we have now.

Reply:
should we just remove it? 

https://github.com/facebook/docusaurus/pull/3800